mne/minimum_norm/resolution_matrix.py

```diff
@@ -424,7 +512,6 @@ def _prepare_info(inverse_operator):
     with info._unlock():
         info["sfreq"] = 1000.0  # necessary
         info["projs"] = inverse_operator["projs"]
-        info["custom_ref_applied"] = False
```

Comment:
I'm not sure we can simply delete this. I think this was somehow necessary to perform EEG source localization.

Reply:
I don't remember doing this.